R/plots-plate.R

```diff
@@ -1,18 +1,18 @@
-#' Plot a 96-well plate with colored wells
+#' Plot a 96-well plate with coloured wells
 #'
-#' It is a generic function to plot a 96-well plate with colored wells
+#' It is a generic function to plot a 96-well plate with coloured wells
 #' used by other functions in this package, mainly to plot layout and counts.
 #' The function uses a background image of a 96-well plate and
-#' plots the colors in the wells using ggplot2.
-#' This function is not intended to be used directly by the user.
-#' Rather, it is used by other functions, specified in this file.
+#' plots the colours in the wells using ggplot2.
+#' This function is not intended for the user to use directly.
+#' Rather, it is used by other functions specified in this file.
 #'
-#' @param colors A vector with 96 colors that will be used to color the wells, order is from left to right and top to bottom
+#' @param colours A vector with 96 colours will be used to colour the wells; the order is from left to right and top to bottom
 #' @param plot_numbers Logical value indicating if the well numbers should be plotted, default is `FALSE`
-#' @param numbers An optional vector with 96 numbers, plotted on the wells. Order is from left to right and top to bottom, must have the same length as colors. It could be used, for instance, to plot the bead count of each well. Must be provided in case the `plot_numbers` parameters is set to `TRUE`
+#' @param numbers An optional vector with 96 numbers plotted on the wells. Order is from left to right and top to bottom and must have the same length as colours. It could be used, for instance, to plot the bead count of each well. Must be provided in case the `plot_numbers` parameters are set to `TRUE`
```

Comment:
Here I think "is" was correct but noun should not be plural

Reply:
changed to 
```

#' @param numbers An optional vector with 96 numbers plotted on the wells. Order is from left to right and top to bottom and must have the same length as colours. 
#' It could be used, for instance, to plot the bead count of each well. Must be provided in case the `plot_numbers` parameter is set to `TRUE`
```